src/manifest.rs

```diff
@@ -148,6 +148,13 @@ pub fn get_crate_name(path: &str) -> Result<String, Error> {
     Ok(read_cargo_toml(path)?.package.name)
 }
 
+pub fn get_wasm_bindgen_version(path: &str) -> Result<Option<String>, Error> {
+    let version: Option<String> = read_cargo_toml(path)?
```

Comment:
you shouldn't need to specify `Option<String>` here. The type is inferred as the return type